patches/chrome-android-java-AndroidManifest.xml.patch

```diff
@@ -12,10 +12,11 @@ index 4b942ff06700f6d8c6ccc1cb94878de7c7412d0d..e1d72e7237d4cf8cabf0974fbea0c55d
      <uses-permission-sdk-23 android:name="android.permission.REORDER_TASKS"/>
      <uses-permission-sdk-23 android:name="android.permission.REQUEST_INSTALL_PACKAGES"/>
  
-@@ -48,9 +50,13 @@ by a child template that "extends" this file.
+@@ -48,9 +50,14 @@ by a child template that "extends" this file.
      {% if target_sdk_version|int > 27 or target_sdk_version == "Q" %}
          <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
      {% endif %}
++    {% if target_sdk_version|int > 29 or target_sdk_version == "R" %} <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES"/> {% endif %}
```

Comment:
@SergeyZhukovsky - Looks like this issue was fixed yesterday in Chromium: https://chromium.googlesource.com/chromium/src/+/aabc32f9845740c239624bdfc4eaedf43e184b60 - I think it might make sense to pull this commit as is to reduce the rebase workload during CR bump. cc: @mkarolin

Reply:
@mkarolin will it help on rebasing? As we have it patched already and strings are not the same as in the upstream commit in any case.

Reply:
nvm I put it at the same position how it's in the upstream